cpp/src/utils/metrics/log_metric.hh

```diff
@@ -0,0 +1,33 @@
+#ifndef TS_CPP_UTILS_METRICS_LOG_METRIC_HH_
+#define TS_CPP_UTILS_METRICS_LOG_METRIC_HH_
+
+#include <string>
+#include <tuple>
+#include <vector>
+#include <map>
+
+#include "src/utils/metrics/metric.hh"
+#include "src/utils/metrics/emitter.hh"
+
+namespace torchserve {
+    // Value entry for log metric consisting of <metric_value, timestamp, request_id>
+    typedef std::tuple<double, std::uint64_t, std::string> LogMetricValue;
```

Comment:
nit: `using LogMetricValue = std::tuple<double, std::uint64_t, std::string>` is more modern?

Reply:
Good point. Will update.